File: Difficulty: Easy/Print Alphabets/print-alphabets.java
//{ Driver Code Starts
//Initial Template for Java

import java.io.*;
class GFG{
    
    
// } Driver Code Ends

// User function Template for Java

// Function should print the alphabets
// from c1 to c2 (space separated)
// No need to print the new line
// The new line will be printed by the
// driver's code.

public static void alphabets(char c1, char c2){
    // Iterate through the characters from c1 to c2
    for(char i = c1; i <= c2; i++){
        // Print each character followed by a space
        System.out.print(i + " ");
    }
}



//{ Driver Code Starts.
    
    
    public static void main(String args[])throws IOException{
        BufferedReader read = new BufferedReader( new InputStreamReader(System.in));
        int t = Integer.parseInt(read.readLine());
        while( t-- > 0){
            
            char c1 = read.readLine().trim().charAt(0);
            char c2 = read.readLine().trim().charAt(0);
            
            alphabets(c1, c2);
            System.out.println();
        
System.out.println("~");
}
    }
}
// } Driver Code Ends

File: Difficulty: Easy/Print Alphabets/print-alphabets.py
#{ 
 # Driver Code Starts
#Initial Template for Python 3

# } Driver Code Ends

# User function Template for python3
def alphabets(c1, c2):
    # Generate and print alphabets from c1 to c2 without a new line after printing
    print(" ".join(chr(c) for c in range(ord(c1), ord(c2) + 1)), end="")


#{ 
 # Driver Code Starts.

if __name__ == '__main__':
    t=int(input())
    for _ in range(t):
        
        c1 = input()
        c2 = input()
        
        alphabets(c1,c2)
        print() #This provides new line
        print("~")
# } Driver Code Ends